dfastmi/batch/AreaPlotter.py

```diff
@@ -0,0 +1,388 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright © 2024 Stichting Deltares.
+
+This library is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation version 2.1.
+
+This library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public
+License along with this library; if not, see <http://www.gnu.org/licenses/>.
+
+contact: delft3d.support@deltares.nl
+Stichting Deltares
+P.O. Box 177
+2600 MH Delft, The Netherlands
+
+All indications and logos of, and references to, "Delft3D" and "Deltares"
+are registered trademarks of Stichting Deltares, and remain the property of
+Stichting Deltares. All rights reserved.
+
+INFORMATION
+This file is part of D-FAST Morphological Impact: https://github.com/Deltares/D-FAST_Morphological_Impact
+"""
+
+from abc import ABC
+from pathlib import Path
+from typing import List
+
+import numpy
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+import dfastmi.kernel.core
+import dfastmi.plotting
+from dfastmi.batch.AreaDetector import AreaDetector
+from dfastmi.batch.PlotOptions import PlotOptions
+
+
+class AreaPlotter(ABC):
+    """
+    Abstract class used to plot the area.
+    """
+
+    _total_str: str
+    _area_str: str
+    _positive_up: bool
+
+    def __init__(
+        self,
+        plotting_options: PlotOptions,
+        plot_n: int,
+        area_detector: AreaDetector,
+        xyz_file_location: Path,
+    ):
+        """
+        Arguments
+        ---------
+        plotting_options : PlotOptions
```

Comment:
Arguments are in the wrong order in the docs :D

Reply:
fixed